controller/src/main/java/io/pravega/controller/server/rpc/grpc/GRPCServerConfig.java

```diff
@@ -95,6 +95,8 @@
      */
     Integer getAccessTokenTTLInSeconds();
 
+    boolean isInternalWritesWithReadPermEnabled();
```

Comment:
we should add a comment to explain this config...

Reply:
Sure, I've added a comment to explain it now.

Reply:
perhaps its good to define what we mean by internal streams here.. 
is it streams that system creates for its own uses - controller's streams?
 or is it system streams created to provide user applications some funciton.
even here we have two types of resources -
1. metadata tables
2. readergroup streams and watermark stream

`2` is something that user application has access to while `1` should be completely hidden from them. 

iiuc you are referring to `2`. right?